src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -122,6 +124,11 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
         Private Sub InitializeIconFile()
             GetProjectsAndProvider()
+            Dim PackageIconUrlSet = TryCast(TryGetNonCommonPropertyValue(GetPropertyDescriptor(_packageIconUrlPropName)), String)
+            If (PackageIconUrlSet IsNot Nothing AndAlso PackageIconUrlSet IsNot "") Then
```

Comment:
Use string.IsNullOrEmpty?

Reply:
I agree it'll be more readable 😄 but should I be consistent with how we already validate?

Reply:
I struggle with the same decision all the time, but in this case I'd say string.IsNullOrEmpty would be much better, so if there are other spots that need updating to use that in this file, then so be it. If the cleanup changes get too big as to make the real changes harder to see, then they can always be deferred to a separate PR.

Reply:
Makes sense 👍 thanks David!